### PR TITLE
Updated model diagrams to align with state of specification as of 02-15-24

### PR DESCRIPTION
**Pull request by sbarnum**

### CODE REVIEW

**Review by kestewart — CHANGES_REQUESTED** [review]
On the Core+Software Diagram,   plase remove the WGs and Exmaples part of it.  It contains stale data, and may cause confusion.

**Comment by kestewart** [comment]
Also please add your - "Signed-off-by: Sean Barnum <email>" to the comments (should be on the commit, but that's harder to do).

**Review by goneall — CHANGES_REQUESTED** [review]
A few requested changes

File: model Core+Software.png
[the remark this answers is not in the document]
Comment (kestewart) [reply model Core+Software.png:1]: packageVerificationCodeExcluededFile was merged.

[the remark this answers is not in the document]
Comment (goneall) [reply model Core+Software.png:1]: Never mind - it looks like in the diagram we don't use the xsd types

[the remark this answers is not in the document]
Comment (goneall) [reply model Core+Software.png:1]: Updated in the latest commit

[the remark this answers is not in the document]
Comment (goneall) [reply model Core+Software.png:1]: Updated in the latest commit

[the remark this answers is not in the document]
Comment (goneall) [reply model Core+Software.png:1]: Updated in the latest commit

**Review by kestewart — APPROVED** [review]
Changes look good to me.

Sean reviewed and added the last commit. 

Based on this,  and the other commit from Gary,  I'm going to go ahead and merge these.

**Review by kestewart — CHANGES_REQUESTED** [review]
@goneall - Sean has made it match the text, with his latest commit.   However I think that this was a case where the diagram was correct, and the text should have been fixed.   https://github.com/spdx/spdx-3-model/blob/main/model/Core/Classes/PackageVerificationCode.md should be updated to allow different algorithms to be specified (ie. SHA256),  right now it's specified as just SHA1.    Issue: https://github.com/spdx/spdx-3-model/issues/646 has been opened to track this.

**Review by kestewart — APPROVED** [review]

**Pull request merged**